dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/fat/AcmeCaRestHandlerTest.java

```diff
@@ -139,7 +143,7 @@ public static void afterClass() throws Exception {
 		/*
 		 * Stop the server.
 		 */
-		server.stopServer();
+		server.stopServer("CWPKI2058W"); // TODO -- check failure revocation "soft error" msg
```

Comment:
Did you mean to leave this TODO?

Reply:
Sort of -- I did see this again while running the test. I will open an issue to track separately and remove the todo.